portal-api/src/v1/validation/route-validators/validate-put-portal-facility-amendment-payload.ts

```diff
@@ -0,0 +1,4 @@
+import { createValidationMiddlewareForSchema } from '@ukef/dtfs2-common';
+import { PORTAL_FACILITY_AMENDMENT_USER_VALUES } from '@ukef/dtfs2-common/schemas';
+
+export const validatePutPortalFacilityAmendmentPayload = createValidationMiddlewareForSchema(PORTAL_FACILITY_AMENDMENT_USER_VALUES.partial());
```

Comment:
is the dealId not passed on the body too?

Reply:
Have just changed how this is passed through (was passing through as a query param before, now is being passed through in the body )